Tests/Fixtures/Generated/JSON/inline-swift3-context-all-publicAccess.swift

```diff
@@ -1,4 +1,5 @@
-// Generated using SwiftGen, by O.Halligon — https://github.com/SwiftGen/SwiftGen
+// swiftlint:disable all
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
```

Comment:
How come we didn't have the `swiftlint:disable all` line on this file (and some others apparently) before this PR?

Reply:
Probablby 2 PRs that were developed in parallel and a needed change that got missed 🤷‍♂️